client/me/concierge/cancel/skeleton.js

```diff
@@ -0,0 +1,25 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import { localize } from 'i18n-calypso';
+
+class Skeleton extends Component {
+	render() {
+		const { translate } = this.props;
+		return (
+			<div>
+				<CompactCard> { translate( 'Cancelling your concierge session…' ) } </CompactCard>
```

Comment:
Nitpick — We're capitalizing "Concierge" in other places, so change this to "Cancelling your Concierge session…"

Reply:
Right, updated.